packages/patternfly-react/src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -31,13 +36,19 @@ FieldLevelHelp.propTypes = {
   /** additional fieldlevelhelp classes */
   content: PropTypes.node,
   /** leave popover/tooltip open  */
-  close: PropTypes.string,
+  rootClose: PropTypes.bool,
```

Comment:
Can we leave `close` and use it if it is given and not `rootClose`? Otherwise, this is a breaking change.